superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -120,22 +120,29 @@ SUPERSET_AUTH_LDAP_ALLOW_SELF_SIGNED: False
 SUPERSET_AUTH_LDAP_APPEND_DOMAIN: ""
 SUPERSET_AUTH_LDAP_USERNAME_FORMAT: ""
 SUPERSET_AUTH_LDAP_SEARCH_FILTER: ""
-RVF_APP_NAME: "Dashboard Builder"
+RVF_APP_NAME: "Dashboard"
 RVF_HELP_ENABLED: False
 SUPERSET_SESSION_LIFETIME_SECONDS: "3600" # stale session timeout
+RVF_LOG_FORMAT: "%(asctime)s:%(levelname)s:%(name)s:%(message)s"
 RVF_LOG_LEVEL: "INFO"
-RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60 * 60 * 24 }}"
-RVF_CACHE_TYPE: 'null'
+RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60*60*24 }}"
+RVF_CACHE_TYPE: 'simple' # enables simple caching to postgres
 RVF_ENABLE_CORS: True
 RVF_CORS_OPTIONS: "{}"
 RVF_WALKME_ENABLED: True
+RVF_DEFAULT_RELATIVE_END_TIME: 'now'
```

Comment:
is this a new var introduced to take care of some use case?

Reply:
No, it's there already.